src/main/resources/pdf_templates/ds01.html

```diff
@@ -0,0 +1,209 @@
+{{"<?xml version=\"1.0\" encoding=\"UTF-8\"?>" | unescaped}}
+
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml"
+      xml:lang="en">
+
+<head>
+    <meta content="application/xhtml+xml; charset=UTF-8" http-equiv="content-type"/>
+    <title>DS01</title>
+
+    <style type="text/css">
+        body {
+            font-size: 14px;
+            font-family: 'TransportNewLight-Regular', sans-serif;
+        }
+
+        img {
+            height: 100%;
+            width: 100%;
+        }
+
+        #page-container {
+            width: 600px;
+            margin: 0 auto
+        }
+
+        .upper {
+            height: 280px;
+            float: left;
+        }
+
+        .lower {
+            float: left;
+        }
+
+        .ch-logo {
+            margin-top: 10px;
+            width: 280px;
+            height: 40px;
+            float: left;
+        }
+
+        .application-type {
+            float: right;
+            text-align: right;
+            margin-right: 10px;
+            font-weight: bold;
+            width: 300px;
+        }
+
+        .application-type-big {
+            font-size: 60px;
+        }
+
+        .application-type-small {
+            font-size: 22px;
+        }
+
+        .box {
+            width: 350px;
+            float: right;
+            border: 5px solid black;
+            padding: 10px 15px 5px 15px;
+            text-align: center;
+            font-size: 30px;
+            display: block;
+        }
+
+        .company-data-table {
+            margin-top: 10px;
+            width: 100%;
+            height: 80px;
+            float: left;
+        }
+
+        .header {
+            margin-top: 30px;
+            text-align: center;
+            font-weight: bold;
+            font-size: 20px;
+        }
+
+        .para {
+            text-align: justify;
+            text-justify: inter-word;
+            margin-top: 25px;
+            margin-bottom: 25px;
+            line-height: 18px;
+        }
+
+        .company-data-td {
+            width: 175px;
+        }
+
+        .company-data-tr {
+            height: 25px;
+        }
```

Comment:
I believe that if you add a class to the table element, then this can be:
```
.company-data-table tr {
  height: 25px;
}

.company-data-table tr td {
  width: 175px;
}
```